### ISSUE TITLE
OnNextBatchAsync on TestStream does not have an implementation

### ISSUE THREAD

**Issue by kehigginbotham** [OrleansContrib/OrleansTestKit#59]
Upgraded to 2.3.1 today.  All unit tests that use TestStream now fail, with the following error:

`StackTrace:	
at Orleans.TestKit.Streams.TestStreamProvider.GetStream[T](Guid streamId, String streamNamespace)   
--- End of stack trace from previous location where exception was thrown ---
Result Message:	System.TypeLoadException : Method 'OnNextBatchAsync' in type 'Orleans.TestKit.Streams.TestStream'1' from assembly 'OrleansTestKit, Version=2.3.1.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.`

Using OrleansTestKit 2.3.1, Orleans.Core.Abstractions 2.3.1

Similar to #49 , but the odd thing is that TestStream<T>.OnNextBatchAsync already exists with the same signature in `OrleansTestKit:2.3.1`

**Comment by sigintarag** [comment]
Looks like OrleansTestKit, Version=2.3.1.0 was compiled against Orleans version 2.3.0 and there is a breaking interface change.  

I was able to get tests to run by downgrading my orleans version to 2.3.0

**Comment by seniorquico** [comment]
I have published an updated OrleansTestKit to NuGet that depends on Orleans 2.3.2. I figured it was only a matter of time before we get a GitHub issue identifying some breaking change against 2.3.2.

For now, I'm going to stick to a policy of updating the OrleansTestKit to the latest release of Orleans. That means we don't/won't have a release against Orleans 2.3.1. Ideally, we won't miss any incremental updates. But as this issue demonstrates, it's a definite reality. I'm open to a discussion (preferably in a new GitHub issue) if someone has a good idea on how to better manage this.
